contributors/devel/sig-release/cherry-picks.md

```diff
@@ -31,37 +31,98 @@ branches.
+
+   The team scrubs through incoming cherry picks on at least a weekly basis, daily during
+   burndown ahead of a .0 release.  Ahead of point releases, reminders of the
+   cherry-pick deadline will be sent out to the community.  Cherry-pick PRs are
+   often metered into the release branches to give more deliberate CI signal across
+   changes.  For this reason your cherry-pick must be merge ready ahead of
+   the cherry-pick deadline, but those candidates may be merged during the days
+   between the deadline and release.
+
+   Open cherry-pick PRs which do not land in the current release will
+   automatically be tracked by the team for inclusion in the next patch
+   release.
+
+   If you are concerned about the status of your cherry-pick, err on the
+   side of overcommunicating and reach out to the branch reviewer(s):
+
+   * During code freeze or after code thaw and ahead of a .0 release, to get attention on a cherry-pick by the current
+     release team members see the [appropriate release folder](https://git.k8s.io/sig-release/releases)
+     for the target release's team contact information. You may cc them with
+     `<@githubusername>` on your cherry-pick PR.
+
    * For prior branches, check the [patch release schedule](https://git.k8s.io/sig-release/releases/patch-releases.md), which includes contact information for the patch release team.
 
-## Cherry-pick Review
+Compared to the normal master branch merges leading up to a release, which
+is typically on the order of 1000 pull requests in 3 months, the release
+branches typically see around 200 pull requests across 9 months. This
+amounts to one or two dozen a month on the stable branches versus many
+hundreds a month on master.  This is because there is an order or two of
+magnitude higher scrutiny.  Again the emphasis is on critical bug fixes.
 
-Cherry-pick pull requests have an additional requirement compared to normal pull
-requests.
-They must be approved specifically for cherry-pick by Approvers.
-The [Branch Manager](https://git.k8s.io/sig-release/release-team/role-handbooks/branch-manager) 
-or the [Patch Release Team](https://git.k8s.io/sig-release/release-team/role-handbooks/patch-release-manager)
-are the final authority on removing the `do-not-merge/cherry-pick-not-approved`
-label  and triggering a merge into the target branch.
-
-Cherry-pick pull requests follow the same release note requirements as
-other pull requests, except the release note stanza will auto-populate from
-the master branch pull request from which the cherry-pick originated.  If
-this is unsuccessful the `do-not-merge/release-note-label-needed` label
-will be applied and the cherry-pick author must edit the pull request
-description to [add a release note](https://git.k8s.io/community/contributors/guide/release-notes.md).
+If you are proposing a cherry pick and it is not a clear and obvious
+critical bug fix, please reconsider.  If upon reflection you wish to
```

Comment:
does this doc detail example critical bug fixes? something similar to the `Criticality / Risk / Complexity / Benefit` portion of https://github.com/kubernetes/sig-release/issues/611 might be beneficial to inline here as examples

Reply:
inlined some examples